product_docs/docs/edbcloud/beta/getting_started/01_check_resource_limits.mdx

```diff
@@ -2,22 +2,23 @@
 title: Raising Azure Resource Limits
 ---
 
-By default, Azure sets a very low limit on the number of virtual machines per region and on the number of Public IP Addresses per region that are available in a given subscription. 
+By default, Azure sets a very low limit on the number of virtual machines vCPU cores per region and on the number of Public IP Addresses per region that are available in a given subscription. 
 
-The default [Azure Virtual Machine  limit per region](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/azure-subscription-service-limits#virtual-machines-limits---azure-resource-manager) is set to 20. You need to increase this limit to at least 50 virtual machines per region. EDB Cloud uses ESv3 Series virtual machines. 
+The default [Azure Virtual Machine vCPU cores limit per region](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/azure-subscription-service-limits#virtual-machines-limits---azure-resource-manager) is set to 20. You need to increase the total cores to account for the total number of cores you expect to use in each region and reserve these cores to be used by EDB Cloud. As an example, if you provision the largest virtual machine with high availability this would be 192 cores per region. You also need to ensure the virtual vCPU cores limits to have 8 DSv2 cores reserved for EDB cloud instrastructure for managing EDB clusters purpose.
 
-The default [Public IP address limits](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/azure-subscription-service-limits#publicip-address) to Public IP Addresses Basic and Public IP Addresses Standards is set to 10. You need to increase the limit of `Public IP Addresses - Basic` and `Public IP Addresses - Standard` to 100 for the regions where you plan to deploy your EDB clusters.
+The default [Public IP address limits](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/azure-subscription-service-limits#publicip-address) to Public IP Addresses Basic and Public IP Addresses Standards is set to 10. You need to increase the limit of `Public IP Addresses - Basic` and `Public IP Addresses - Standard` for the regions where you plan to deploy your EDB clusters with the number of how many EDB clusters do you plan to use in that region.
```

Comment:
I would change to:
"You need to increase the limit of `Public IP Addresses - Basic` and `Public IP Addresses - Standard` for the regions where you plan to deploy your EDB clusters with the total number of EDB clusters you plan to use."